crates/sui-core/src/quorum_driver/mod.rs

```diff
@@ -39,6 +42,27 @@ const TASK_QUEUE_SIZE: usize = 10000;
 const EFFECTS_QUEUE_SIZE: usize = 1000;
 const TX_MAX_RETRY_TIMES: u8 = 10;
 
+/// Quorum Driver Internal types
+#[derive(Error, Debug)]
+pub(crate) enum QuorumDriverInternalError {
+    #[error(
+        "Failed to execute transaction on a quorum of validators. Validator errors: {:?}",
```

Comment:
Can we be more precise here, e.g. "Failed to get a quorum of validators to sign the transaction. Validator errors: ..."?